netbox/dcim/views.py

```diff
@@ -3131,6 +3131,13 @@ def get_object(self, **kwargs):
 
         return obj
 
+    def get_extra_addanother_params(self, request):
+        return {
+            'termination_a_device': resolve(request.GET.get('return_url')).kwargs.get('pk'),
```

Comment:
We can't assume the A side will be a device; it could also be a power feed or circuit termination.

Reply:
Hi @jeremystretch, thank you for the review. What should be the correct approach for you ?

1. from the request object, we could return `a_terminations` (in place of `termination_a_device`), despite a connection already exists,
2. parse `a_terminations` and only return the parent object pk (as you mentioned device, power feed or circuit, but with the corresponding field...) and let user choose the termination for A side.

Thanks in advance for your advise.